### ISSUE TITLE
Map doesn't load on iOS 10.3. Black screen

### ISSUE THREAD

**Issue by laurismikals** [nitaliano/react-native-mapbox-gl#562]
# Environment
iOS 10.3, React Native Mapbox GL 5.1.0, React Native 0.43, React 16.0.0-alpha.6
# Problem
After starting the aplication the Mapbox component starts, however map doesn’t load. There is only black screen. Xcode console keeps printing error: “nw_connection_get_connected_socket_block_invoke Connection has no connected handler”. Sometimes after few minutes the map loads.
On iOS 9.3 map works as expected. On iOS 10 it doesn’t work.
# Visual Proof
Video where map doesn’t work on iOS 10.3: [https://youtu.be/Sh93-hr19b0](https://youtu.be/Sh93-hr19b0)
Video where map works on iOS 9.3: [https://youtu.be/A4IAXKxEfM8
](https://youtu.be/A4IAXKxEfM8)
```
export default class baltanakts extends Component{
 render(){
   return(
     <View
       style={styles.container}
     >
       <MapView
         style={styles.map}
         initialCenterCoordinate={{
           latitude: 56.949506,
           longitude: 24.105689
         }}
         initialZoomLevel={11}
         rotateEnabled={false}
         logoIsHidden={true}
       />
     </View>
   )
 }
}
```

**Comment by RobertSasak** [comment]
I have a similar issue where map is black and after a few minutes all the tiles appear. I can see big CPU load in lib related to sql. I subscribed to the `Mapbox.addOfflinePackProgressListener()` and it looks like tiles are redownload again. I cannot see any network traffic, so I assume that Mapbox is just checking tiles in it's sql db. Also more maps I have downloaded for offline usage, longer map show black. Please let me know if your issue can be also related to offline maps. I am running my app on 10.3.2

This issue looks very similar https://github.com/mapbox/react-native-mapbox-gl/issues/525

Update: I followed @butchmarshall advice and used the latest master. Offline maps are still downloaded everytime I start app.
```
"dependencies": {
     ...
    "react-native-mapbox-gl": "git://github.com/mapbox/react-native-mapbox-gl.git#b46ea8da99",
  },
```

**Comment by henrikra** [comment]
@laurismikals Is this solved?

**Comment by RobertSasak** [comment]
For me it was solved when I updated from 5.2.0 to  5.2.3-alpha1.
